src/server.c

```diff
@@ -3720,6 +3725,14 @@ static void handle_join(int cid, char *ptr)
 		send_msgf(cid, MSG_JOINNAK, "s", "Incorrect game password");
 		return;
 	}
+	/* Check for a version compatible with latest expansion */
+	if (s_list[sid].expanded == 5 && strcmp(c_list[cid].version, "0.9.5") < 0)
+	{
+		/* Send denied message */
+		send_msgf(cid, MSG_JOINNAK, "s", "Client version does not support Xeno"
```

Comment:
Shall we add an issue reminding us to add the relevant information to the wiki before the release?